.github/workflows/ai-pr-review.yml

```
@@ -0,0 +1,149 @@
+name: AI PR Review
+
+on:
+  pull_request:
+    types: [review_requested]
+  workflow_dispatch:
+    inputs:
+      pr_number:
+        description: 'Pull request number to review'
+        required: true
+        type: number
+
+jobs:
+  check-conditions:
+    runs-on: ubuntu-24.04
+    outputs:
+      should_review: ${{ steps.set-output.outputs.should_review }}
+    steps:
+      - name: Set trigger type
+        id: trigger-type
+        env:
+          EVENT_NAME: ${{ github.event_name }}
+        run: |
+          if [[ "$EVENT_NAME" == "workflow_dispatch" ]]; then
+            echo "is_workflow_dispatch=true" >> $GITHUB_OUTPUT
+          else
+            echo "is_workflow_dispatch=false" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Check branch name
+        id: check-branch
+        if: steps.trigger-type.outputs.is_workflow_dispatch != 'true'
```

Can't you just do this?
```suggestion
        if: github.event_name != "workflow_displatch"
```

---

Probably, the AI wrote these. It's working so I'd rather not touch it